src/scss/components/box/_mixins.scss

```diff
@@ -0,0 +1,12 @@
+@import "vars";
+
+// less: #Box.xs()    -> box(xs)
+// less: #Box.sm()    -> box(sm)
+// less: #Box.md()    -> box(md) or box() or box [DEFAULT]
+// less: #Box.lg()    -> box(lg)
+// less: #Box.xl()    -> box(xl)
+// less: #Box.xxl()   -> box(xxl)
+// less: #Box.flush() -> box(flush)
+@mixin box($size: md) {
+  padding: map-get($space, $size);
+}
```

Comment:
SASS allowed us to simplify the mixins by pulling directly from the global `$space` map (variable).

Reply:
Changes to Box mixins will be handled by #588.